scripts/deploy/aws.sh

```diff
@@ -111,6 +111,8 @@ function render_workspace() {
 
 # This script is designed to work in the project root
 cd "$PROJECT_ROOT"
+allowed=("--help" "--console" "--check" "--render" "--deploy-eks" "--deploy-x1" "--config" "--delete" "--delete-x1" "--delete-eks" "--start-proxy" "--stop-proxy")
+check_args $@ allowed 
```

Comment:
`allowed` is a global variable, no need to pass. Also passing like this does not work, recommended change:
```suggestion
check_args "$@"
```